README.md

```diff
@@ -35,10 +50,22 @@ For better performance, build examples with the [release compilation profile](ht
 cargo run --release --example <example_name>
 ```
 
+## Guides
+
+Step-by-step introduction guides are available on the [LiveCompositor website](https://compositor.live/docs/guides).
+
 ## Supported platforms
 
 Linux and macOS.
 
+## Current development
```

Comment:
I wouldn't mention that in readme. If we had some specific plans or some formalized roadmap then it would make sense, but this does not look good.